VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -101,65 +101,62 @@
 
             <LinearLayout
                 android:id="@+id/last_access_bar"
-                android:orientation="vertical"
                 style="@style/last_access_bar_style"
-                android:visibility="gone">
```

Comment:
> The view's visibility is set to gone in the last_access_bar_style, thats why I removed it from here.

My bad, I didn't even look there. I guess I assumed it wouldn't be there because toggling visibility isn't really "styling". Anyway that's fine :+1: